src/semver.toit

```diff
@@ -117,6 +119,81 @@ compare a/string b/string [--if_equal]:
   if comp != 0: return comp
   return if_equal.call
 
+is_letter_ c/int -> bool:
+  return 'A' <= c <= 'Z' or 'a' <= c <= 'z'
+
+is_digit_ c/int -> bool:
+  return '0' <= c <= '9'
+
+is_non_digit_ c/int -> bool:
```

Comment:
`is_valid_non_digit_`?

Reply:
These names come from the semver specification.
I prefer to keep them the way they are.